diff --git a/node_modules/aws-serverless-express/src/index.js b/node_modules/aws-serverless-express/src/index.js
index 3db57d3..856d6ea 100644
--- a/node_modules/aws-serverless-express/src/index.js
+++ b/node_modules/aws-serverless-express/src/index.js
@@ -19,7 +19,7 @@ const binarycase = require('binary-case')
 const isType = require('type-is')
 
 function getPathWithQueryStringParams (event) {
-  return url.format({ pathname: event.path, query: event.queryStringParameters })
+  return url.format({ pathname: event.path, query: event.multiValueQueryStringParameters })
 }
 function getEventBody (event) {
   return Buffer.from(event.body, event.isBase64Encoded ? 'base64' : 'utf8')
